castle/src/main/java/io/castle/android/Utils.java

```diff
@@ -17,17 +17,17 @@
 import java.util.Locale;
 import java.util.TimeZone;
 
+import io.castle.android.api.model.Custom;
```

Comment:
Should this be called `CustomEvent` for consistency?

Reply:
Good point, let's do that